pandas/core/generic.py

```diff
@@ -10329,6 +10329,28 @@ def _doc_parms(cls):
 %(examples)s
 """
 
+_num_doc_mad = """
+%(desc)s
+
+Parameters
+----------
+axis : %(axis_descr)s
+    Axis for the function to be applied on.
+skipna : bool, default True
+    Exclude NA/null values when computing the result.
+level : int or level name, default None
+    If the axis is a MultiIndex (hierarchical), count along a
+    particular level, collapsing into a %(name1)s.
+**kwargs
+    Additional keyword arguments to be passed to the function.
```

Comment:
Does this function accept kwargs?
```
def mad(self, axis=None, skipna=None, level=None):
```

Reply:
I don't think so. I will remove this as well.

Reply:
I've updated the commit. Please review it!